charts/sn-platform/templates/broker/broker-cluster.yaml

```diff
@@ -195,9 +195,34 @@ spec:
     {{- end }}
     {{- end }}
     custom:
+      ## Offloading settings
+      {{- if .Values.broker.offload.enabled }}
+      offloadersDirectory: "{{ template "pulsar.home" . }}/offloaders"
+      {{- if .Values.broker.offload.use }}
+      managedLedgerOffloadDriver: {{ .Values.broker.offload.managedLedgerOffloadDriver }}
```

Comment:
I have a question. When gcs or s3 is enabled, they both can use the same offload driver `aws-s3`, right?
I saw the example driver of gcs can be `google-cloud-storage`, and aws can be `aws-s3` or `s3`
[configure-gcs-offloader-driver](https://pulsar.apache.org/docs/en/tiered-storage-gcs/#configure-gcs-offloader-driver)
[configure-aws-s3-offloader-driver](https://pulsar.apache.org/docs/en/tiered-storage-aws/#configure-aws-s3-offloader-driver)

Reply:
.Values.broker.offload.managedLedgerOffloadDriver can be user provided, it doesn't have to be aws-s3 right?